### ISSUE TITLE
Regarding training my customized dataset

### ISSUE THREAD

**Issue by liuch37** [OpenBMB/VisRAG#52]
Hi,

Thank you for your great work! I would like to know if I want to create my own customized dataset to train VisRAG-Ret, what should I put in my folder? Could you provide an example folder for a customized training dataset?

Thanks a lot!

**Comment by tcy6** [comment]
@liuch37 No problem!

I've uploaded an example folder to this repository. As you can see, your training data should be formatted in `parquet` format, with two key fields: `query` and `image`. The `query` field should be of type `str`, while the `image` field should be of type `bytes`.

Additionally, you should include a `metadata.json` file in the folder, as shown below. The `length` field indicates the number of training examples, which ensures the proper functioning of the learning rate scheduler:

```json
{
  "length": 4096
}
```
⚠️ Important: If you wish to train using your own datasets, don't forget to remove the `--from_hf_repo` line from the `train.sh` script.
